docs/outputs.rst

```diff
@@ -285,16 +288,36 @@ the command line options ``--fd-spike-threshold`` and ``--dvars-spike-threshold`
 (default: FD > 0.5 mm or DVARS > 1.5).
 Spike regressors are stored in separate ``motion_outlier_XX`` columns.
 
-**AROMA confounds**.
-:abbr:`AROMA (Automatic Removal Of Motion Artifacts)` is an :abbr:`ICA (independent components analysis)`
-based procedure to identify confounding time series related to head-motion [Prium2015]_.
-ICA-AROMA can be enabled with the flag ``--use-aroma``.
-
-- ``aroma_motion_XX`` - the motion-related components identified by ICA-AROMA.
-
-.. danger::
-    If you are already using ICA-AROMA cleaned data (``~desc-smoothAROMAnonaggr_bold.nii.gz``),
-    do not include ICA-AROMA confounds during your design specification or denoising procedure.
+**Discrete cosine-basis regressors**.
+Physiological and instrumental (scanner) noise sources are generally present in fMRI
+data, typically taking the form of low-frequency signal drifts.
+To account for these drifts, temporal high-pass filtering is the immediate option.
+Alternatively, low-frequency regressors can be included in the statistical model to account
+for these confounding signals.
+Using the :abbr:`DCT (discrete cosine transform)` basis functions, *fMRIPrep* generates
+these low-frequency predictors:
+
+- ``cosine_XX`` - DCT-basis regressors.
+
+One characteristic of the cosine regressors is that they are identical for two different
+datasets with the same :abbr:`TR (repetition time)` and the same *effective* number of
+time points (*effective* length).
+It is relevant to mention *effective* because initial time points identified as *nonsteady
+states* are removed before generating the cosine regressors.
+
+.. caution::
+    If the analysis includes your own high-pass filtering step, the ``cosine_XX``
+    regressors may interfere (meaning, they may re-introduce drifts) if included
+    in the design matrix.
```

Comment:
I'm a little skeptical of this claim of reintroducing drifts.

What about:

"If your analysis includes separate high-pass filtering, do not also include ``cosine_XX`` regressors in your design matrix."